### PR TITLE
AO3-4991 Reorder series page navigation buttons

### PR DESCRIPTION
**Pull request by niconicosette**
## Issue

https://otwarchive.atlassian.net/browse/AO3-4991

## Purpose

move `edit series` and `delete series` buttons to top of series pages.

## Testing

confirm that the `edit series` and `delete series` buttons are now cozy up at the top with the other series navigation buttons.

### CODE REVIEW

**Comment by redsummernight** [comment]
This PR has unrelated changes from [AO3-3507](https://otwarchive.atlassian.net/browse/AO3-3507).

File: app/views/series/_series_navigation.html.erb [hunk 1]
```diff
@@ -16,4 +8,12 @@
       <li><%= link_to ts("Remove Me As Author"), edit_series_path(@series, :remove => 'me'), data: {confirm: series_removal_confirmation(@series, current_user)} %></li>
     <% end %>
   <% end %>
+  <li><%= span_if_current ts("Series"), series_path(@series) %></li>
+  <% if @subscription %>
+    <li>
+      <%= render :partial => 'subscriptions/form', :locals => { :subscription => @subscription } %>
```
Comment (sarken) [inline app/views/series/_series_navigation.html.erb]: Could you update the syntax and switch to double quotes here? `<%= render "subscriptions/form", subscription: @subscription %>`

**Comment by niconicosette** [comment]
were the syntax changes done correctly? 💦

**Comment by sarken** [comment]
Yup, the syntax changes look great! Travis being a little peculiar about a seemingly unrelated test, but I've restarted it, and assuming it passes, I'll mark this ready to merge :+1:

**Pull request merged**